### ISSUE TITLE
Input value will undo on focusout when using only Ctrl+Backspace

### ISSUE THREAD

**Issue by camilova** [autoNumeric/autoNumeric#656]
### Current behavior
If you use the next options on initialization with version `4.5.8`:

`autonumeric_options = {
    digitGroupSeparator: '.',
    decimalCharacter: ','
  };`

And you focus the input and hit `CTRL+BACK` or `CTRL+SUPR` to clear the value quickly, then after focusout **the change will undo**, showing the last value.

### Expected behavior
The input should be empty after you press `CTRL+SUPR` and `focusout`.

### Steps to reproduce the problem
- Initialize a simple input with the options:

```
autonumeric_options = {
  digitGroupSeparator: '.',
  decimalCharacter: ','
};
AutoNumeric.multiple("input", autonumeric_options);
```
- Focus the input
- Press `CTRL+BACK`
- Click on any place to `focusout` the input or press `TAB`

[Here is a example showing the behavior](https://jsfiddle.net/r7vhpyoL/2/)

This behavior happens on **Chrome**, but in **Firefox** the value is not cleared at all.

**Comment by AlexandreBonneau** [comment]
Thanks for reporting this.
I'm astonished I never stumbled upon that buggy behavior before!

**Comment by camilova** [comment]
I’m happy to help, thanks for the useful library!

**Comment by AlexandreBonneau** [comment]
This is fixed in `4.5.10`.
